apps/evm/lib/evm/builtin.ex

```diff
@@ -41,17 +42,12 @@ defmodule EVM.Builtin do
   @spec run_ecrec(EVM.Gas.t(), EVM.ExecEnv.t()) ::
           {EVM.Gas.t(), EVM.SubState.t(), EVM.ExecEnv.t(), EVM.VM.output()}
   def run_ecrec(gas, exec_env = %EVM.ExecEnv{data: data}) do
-    used_gas = @g_ecrec
+    required_gas = @g_ecrec
```

Comment:
I like this rename. Since we haven't "used" the gas yet it makes sense that is is how much is required for the work we'd like to do. 👍